### PR TITLE
DR-962: Add log sink and BQ view for PerformanceLogger statements.

### PR DESCRIPTION
**Pull request by marikomedlock**
Copied the code for the existing user_activity log sink and corresponding BQ structured log parsing view, and modified for the PerformanceLogger statements.

### CODE REVIEW

File: bq.tf.ctmpl [hunk 1]
```diff
@@ -22,6 +22,21 @@ SELECT timestamp,
   REGEXP_EXTRACT(textPayload, r"method: ([^,]+)") AS method,
   REGEXP_EXTRACT(textPayload, r"url: ([^,]+)") AS url
 FROM `${data.google_project.project.name}.${module.user-activity-sinks.dataset_id[0]}.stdout_*`;
+EOF
+    use_legacy_sql = false
+  }
+
+  view {
```
Comment (fboulnois) [inline bq.tf.ctmpl:29]: I think this needs to be in a separate resource block as it will need a unique `table_id`

File: variables.tf.ctmpl [hunk 2]
```diff
@@ -104,6 +104,16 @@ variable "user_activity_enable_bigquery" {
   default = "{{ .Data.user_activity_enable_bigquery }}"
 }
 
+variable "performance_log_enable_pubsub" {
+  type = number
+  default = "{{ .Data.performance_log_enable_pubsub }}"
```
Comment (fboulnois) [inline variables.tf.ctmpl:109]: I don't think these defaults are defined anywhere, hence the `terraform init` failure.

File: bq.tf.ctmpl (hunk 1 above)
Comment (marikomedlock) [reply bq.tf.ctmpl:38]: Ah okay, that makes sense. Trying that... Thank you!

File: variables.tf.ctmpl (hunk 2 above)
Comment (marikomedlock) [reply variables.tf.ctmpl]: Yep, good catch. @smark88 says this variable defaulting was a way of differentiating between dev/prod/integration, and that this approach is going away soon.

I think we don't need to make this new BQ table/view environment dependent anyway. We may decide not to use it for prod, and then it'd be empty, but I'd like to leave that possibility open. Other projects I've been on have done periodic/limited performance logging on prod for diagnostic purposes.

**Pull request closed without merging**